test/embed.test.js

```diff
@@ -81,31 +81,31 @@ describe('Embed Tests', () => {
   });
 
   it('Response is cacheable', async () => {
-    const { headers, body } = await embed('http://httpbin.org');
+    const { headers, body } = await embed('http://httpbin.org', { kind: 'embed-httpbin' });
     assert.equal(headers['Cache-Control'], 'max-age=3600');
     assertContains(body, ['http://httpbin.org']);
   });
 
   it('Response is HTML', async () => {
-    const { headers, body } = await embed('https://www.npmjs.com/package/@adobe/helix-cli');
+    const { headers, body } = await embed('https://www.npmjs.com/package/@adobe/helix-cli', { kind: 'embed-npmjs' });
     assert.equal(headers['Content-Type'], 'text/html');
     assertContains(body, ['https://www.npmjs.com/package/@adobe/helix-cli']);
   });
 
   it('Supports Image Cards', async () => {
-    const { headers, body } = await embed('https://blog.twitter.com/en_us/a/2015/history-of-tbt-on-twitter.html');
+    const { headers, body } = await embed('https://blog.twitter.com/en_us/a/2015/history-of-tbt-on-twitter.html', { kind: 'embed-twitter-blog' });
     assert.equal(headers['Content-Type'], 'text/html');
     assertContains(body, ['https://blog.twitter.com/en_us/a/2015/history-of-tbt-on-twitter.html']);
   });
 
   it('Supports Images', async () => {
-    const { headers, body } = await embed('https://unsplash.com/photos/VS_kFx4yF5g');
+    const { headers, body } = await embed('https://unsplash.com/photos/VS_kFx4yF5g', { kind: 'embed-unsplash' });
     assert.equal(headers['Content-Type'], 'text/html');
     assertContains(body, ['embed-has-image']);
   });
 
   it('Supports Adobe Spark', async () => {
-    const { headers, body } = await embed('https://spark.adobe.com/post/z4eHLkF8nZII1/');
+    const { headers, body } = await embed('https://spark.adobe.com/post/z4eHLkF8nZII1/', { kind: 'embed-spark-adobe' });
```

Comment:
why don't you test for `embed-spark-adobe` ?

Reply:
Shouldn't it be `embed-adobe-spark`? And does it have an additional class `embed-adobe` class as suggested in a previous comment?